app/views/comments/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Comments</h1>
+<h1>Your Comments</h1>
```

Comment:
Aren't we supposed to allow unauthenticated users to see all comments still?

Reply:
The /comments is just for logged in users to see all their comments. 
At least that's my interpretation of it. 
Unauthenticated users can still view comments made on posts by viewing a post